src/api.rs

```diff
@@ -69,3 +69,24 @@ pub fn listit() -> Result<Vec<(String, TimestampWithTimeZone)>, spi::Error> {
     });
     Ok(results)
 }
+
+#[pg_extern]
+fn pgmq_send_batch(
+    queue_name: &str,
+    messages: Vec<pgrx::JsonB>,
+    delay: default!(i64, 0),
```

Comment:
```suggestion
    delay: default!(i64, 0)
```
I guess we will add `delay` as an optional argument for `pgmq_send` as well?

Reply:
Yes, thought about doing that in this PR but we can do it separately.